selendroid-standalone/src/test/java/io/selendroid/standalone/server/model/DeviceStoreFixture.java

```diff
@@ -32,6 +34,7 @@ protected static DefaultHardwareDevice anDevice(String name, DeviceTargetPlatfor
     DefaultHardwareDevice device = mock(DefaultHardwareDevice.class);
     when(device.getModel()).thenReturn(name);
     when(device.getTargetPlatform()).thenReturn(platform);
+    when(device.getModel()).thenReturn("Nexus 5");
```

Comment:
The default value for this will be an empty string? If yes, can you change this to return an empty string instead and explicitly set the value to "Nexus 5" for the tests where you need that?